.gitignore

```diff
@@ -132,6 +132,10 @@ fixtures/app/extensions/*/build
 .yarn*
 .pnp.*
 
+# embed CLI2
+Gemfile
+README.md
```

Comment:
For any reason, when you run the tests, those file are generated at the root of the project. Maybe this is the source of the problem of why unit tests are not working, because their configuration does not detect the root folder properly. 
Once I fix that problem I will remove these entries

Reply:
I'm confused because we already have a README.md in the root. And if we merge this, any change in that file would be ignored, wouldn't it?